policy/modules/contrib/networkmanager.te

```diff
@@ -33,6 +33,13 @@ files_type(NetworkManager_var_lib_t)
 type NetworkManager_var_run_t;
 files_pid_file(NetworkManager_var_run_t)
 
+type NetworkManagerDispatcher_t;
+type NetworkManagerDispatcher_exec_t;
+init_daemon_domain(NetworkManagerDispatcher_t, NetworkManagerDispatcher_exec_t)
+
+type NetworkManagerDispatcher_initrc_exec_t;
+init_script_file(NetworkManagerDispatcher_initrc_exec_t)
```

Comment:
Since the initrc concept is kind of legacy, I think the string and `init_script_file()` should not be used. It'd be fine have just lib_t for /usr/lib, but some label needs to be set for /etc, too.

Reply:
It was set more clear label for scripts **NetworkManager_dispatcher_script_t**.